### PR TITLE
Setup jaeger operator

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I added manifest to deploy jager-operator and jaeger.

The following file is a custom resource for deploying jager.
- https://github.com/vdaas/vald/blob/feature/setup-jaeger-operator/k8s/metrics/jaeger/jaeger/jaeger.yaml

### Related Issue:

Nothing

### Environment:

- Go Version: 1.18.2
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.5

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hlts2** [comment]
I have completed testing it locally.
It was working fine.
image

------

image

**Review by datelier — APPROVED** [review]

File: Makefile.d/k8s.mk [hunk 1]
```diff
@@ -211,12 +211,17 @@ k8s/metrics/grafana/delete:
 .PHONY: k8s/metrics/jaeger/deploy
 ## deploy jaeger
 k8s/metrics/jaeger/deploy:
-	kubectl apply -f k8s/metrics/jaeger
+	kubectl apply -f https://raw.githubusercontent.com/jaegertracing/helm-charts/jaeger-operator-2.30.0/charts/jaeger-operator/crds/crd.yaml
```
Comment (kpango) [inline Makefile.d/k8s.mk:214]: can you define operator version as variable?

Comment (hlts2) [reply Makefile.d/k8s.mk]: I have fixed :)
https://github.com/vdaas/vald/pull/1682/commits/65e5e53734d5cfb9b6f717cef48fa693b804456f

**Review by kevindiu — APPROVED** [review]
LGTM

**Comment by hlts2** [comment]
@kpango @kevindiu 
Thank you for your review 🙏 
I will merge this PR.

**Pull request merged**